blog/exporters.md

```diff
@@ -1,19 +1,22 @@
 ---
-title: How to use Security Advisor with Skydive
+title: How to use Exporter pipeline with Skydive
 layout: blog-post
 author: Kalman Meth
 date: 31/07/2019
 ---
 
+The Exporter filters the flow data obtained from Skydive, performs a data transformation, and saves the information to some target (e.g. an object store).
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term `pipeline` to describe the sequence of operations performed on the data: `classify`, `filter`, `transform`, `encode`, `store`.
+The combinations of filter, transformation, encoder, etc, allow the exporter to be exploited for different types of applications.
+In addition to the core Exporter modules, we have a Security Advisor application that is built using the Exporter components.
+Other applications based on the Exporter pipeline components are expected to be made available in the near future.
 
 The Security Advisor filters the flow data obtained from Skydive, performs a data transformation, and saves the information to an object store in GZIP compressed JSON encoding format.
-This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
 
-Instructions to developers to deploy and run the Secruity Advisor can be found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor/README.md).
+Instructions to developers to deploy and run the Secruity Advisor can be found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/exporters/secadvisor/README.md).
```

Comment:
Why "secadvisor" in the path ... I thought we move secadvisor to be only configuration ... in any event the README.md should moveto https://github.com/skydive-project/skydive/tree/master/contrib/exporters/ level @hunchback ?

Reply:
for now the README.md is specific to secadvisor, but I agree - it should be generalized and moved one level up in the future (and then we can update the link here)